### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by spetterman66**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -72,7 +75,8 @@ Here's an overview of the operating systems supported in v86:
   - [SkiffOS](https://github.com/skiffos/SkiffOS/tree/master/configs/browser/v86) (based on Buildroot) can cross-compile a custom image.
   - Archlinux works. See [archlinux.md](docs/archlinux.md) for building an image.
   - Debian works. An image can be built from a Dockerfile, see [tools/docker/debian/](tools/docker/debian/).
-  - Ubuntu up to 16.04 works.
+  - Ubuntu up to 16.04 LTS works.
+  - Lubuntu/Ubuntu Budgie/Kubuntu up to 18.04 LTS
```
Comment (copy) [inline Readme.md:79]: Does regular ubuntu 18.04 work? I'd compress this to "Ubuntu up to 18.04 LTS works." and remove the unnecessary mentions of all the ubuntu derivatives.

File: Readme.md [hunk 2]
```diff
@@ -82,6 +86,7 @@ Here's an overview of the operating systems supported in v86:
   versions may work if compiled without SSE3. See [#224](https://github.com/copy/v86/issues/224).
 - Windows 1, 3.0, 95, 98, ME and 2000 work. Other versions currently don't (see [#86](https://github.com/copy/v86/issues/86), [#208](https://github.com/copy/v86/issues/208)).
   - In Windows 2000 and higher the PC type has to be changed from ACPI PC to Standard PC
+- Windows 8 build 8133 works
```
Comment (copy) [inline Readme.md:89]:
```suggestion
- Windows 1, 3.x, 95, 98, ME and 2000 work reasonably well
  - In Windows 2000 and higher the PC type has to be changed from ACPI PC to Standard PC
  - There are some known boot issues (#250, #433, #507, #555, #620, #645)
- Windows XP, Vista and 8 work under certain conditions (see [#86](https://github.com/copy/v86/issues/86), [#208](https://github.com/copy/v86/issues/208))
```

File: Readme.md (hunk 1 above)
Comment (SuperMaxusa) [reply Readme.md:79]: In official Ubuntu, the latest x86 version became [16.04](https://releases.ubuntu.com/16.04/), but in some distributions 32-bit versions remained until 18.04 (for example, [Kubuntu](https://cdimage.ubuntu.com/kubuntu/releases/18.04.5/release/))

Comment (spetterman66) [reply Readme.md:79]: in the case of Lubuntu, 18.10 (but it's not supported anymore)

Comment (spetterman66) [reply Readme.md:79]: Same with Ubuntu Server 17.10 (see http://old-releases.ubuntu.com/releases/17.10/)

**Pull request merged**